### ISSUE TITLE
Cannot start HomeBridge

### ISSUE THREAD

**Issue by luisiam** [AlexGustafsson/homebridge-wol#2]
/usr/local/lib/node_modules/homebridge/lib/api.js:47
      throw new Error("The requested accessory '" + name + "' was not registered by any plugin.");
      ^

Error: The requested accessory 'Computer' was not registered by any plugin.
    at API.accessory (/usr/local/lib/node_modules/homebridge/lib/api.js:47:13)
    at Server._loadAccessories (/usr/local/lib/node_modules/homebridge/lib/server.js:164:42)
    at Server.run (/usr/local/lib/node_modules/homebridge/lib/server.js:36:38)
    at module.exports (/usr/local/lib/node_modules/homebridge/lib/cli.js:23:16)
    at Object.<anonymous> (/usr/local/lib/node_modules/homebridge/bin/homebridge:17:22)
    at Module._compile (module.js:398:26)
    at Object.Module._extensions..js (module.js:405:10)
    at Module.load (module.js:344:32)
    at Function.Module._load (module.js:301:12)
    at Function.Module.runMain (module.js:430:10)
    at startup (node.js:141:18)
    at node.js:980:3

**Comment by AlexGustafsson** [comment]
I'm aware of this issue and I will try to fix it ASAP today. Thank you for letting me know.

**Comment by AlexGustafsson** [comment]
The cause of the error was an error in the plugin. I was working on some new features which did not seem to work as planned. The project has now been split into two branches, master and stable. The stable is the one published on npm.

TLDR;
Run `npm install -g homebridge-wol` to be able to use it as intended.
